src/dotnet/CarbonAware.WebApi/Program.cs

```diff
@@ -7,6 +9,10 @@
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
+// Load configuration file then Register CarbonAware Services using Extension.
+var configuration = new ConfigurationBuilder().AddJsonFile("appsettings.json").Build();
```

Comment:
This line shouldn't be necessary, as `appsettings.json` should be picked up automatically and can be referenced via `builder.Configuration`

Our current launch configuration needs to be updated for the above to work though. It's fixed in [this change](https://github.com/microsoft/carbon-aware-sdk/pull/11/files#diff-bd5430ee7c51dc892a67b3f2829d1f5b6d223f0fd48b82322cfd45baf9f5e945R14).  Feel free to pull in those files and we can race to merge ;)

Reply:
Configuration arg removed from extension for now.